Main.cpp

```diff
@@ -168,6 +155,43 @@ Export int InitProc()
 	return 1; // return 1 if OK; 0 on failure
 }
 
+void SetDifficultyMultiplier()
+{
+	switch (Player[0].Difficulty())
+	{
+	case DiffEasy: {
+		saveData.diffMultiplier = 13;
+		return; }
+	case DiffNormal: {
+		saveData.diffMultiplier = 10;
+		return; }
+	case DiffHard: {
+		saveData.diffMultiplier = 7;
+		return; }
+	}

```

Comment:
The brace formatting looks a bit messy. The indentation looks a bit excessive (might be a GitHub display thing).

----

For an alternative to `switch`, you can also use a small lookup table:
```cpp
const int diffMultipliers[] = {13, 10, 7}; // Easy, Medium, Hard
return diffMultipliers[Player[0].Difficulty()];
```

Reply:
Good idea.